.github/CODEOWNERS

```diff
@@ -1,4 +1 @@
-# Thanks to @imbstack for the suggestion.
-# see http://code.v.igoro.us/posts/2019/07/codeowners.html
-
-* @mozilla-it/syseng
+* @robhudson
```

Comment:
I think we can also add `@mozmeao/backend` to include the 3 of us.

Reply:
Will do!

Reply:
Is that a team in https://github.com/orgs/mozilla-it/? All I was able to find was https://github.com/orgs/mozilla-it/teams/mozmeao. I don't think you can do cross-org codeownership like that, unfortunately

Reply:
Okay, makes sense. We could use the `@mozmeao` one. I think it would be nice to have a "main" but also a team as fallback.